templates/index.html

```diff
@@ -38,390 +41,523 @@ <h2 class="p-heading--two">Say hello to your physical&nbsp;cloud</h2>
   </div>
 </section>
 
-<section class="p-strip p-strip--light is-deep">
+<section class="p-strip--light">
   <div class="row">
-    <div class="col-5">
-      <h2 class="u-no-margin--bottom">The smartest way to manage bare metal</h2>
+    <h3 class="p-muted-heading u-align-text--center">A selection of customers</h3>
+    <ul class="u-no-margin">
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}d8f890fb-logo-at%26t.svg" alt="AT&T logo" />
+      </li>
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}4589dd76-logo-deutsche-telekom.svg" alt="Deutsche Telekom logo" />
+      </li>
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}c15698ab-logo-microsoft.svg" alt="Microsoft logo" />
+      </li>
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}e4aa11e6-logo-nec.svg" alt="NEC logo" />
+      </li>
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}cd22a35e-logo-ntt.svg" alt="NTT logo" />
+      </li>
+      <li class="mobile-col-2 tablet-col-2 col-2">
+        <img src="{{ ASSET_SERVER_URL }}cba50650-logo-verizon.svg"  alt="Verizon logo" />
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Real servers.<br />Self service.</h2>
     </div>
-    <div class="col-7 prefix-1">
-      <p class="p-heading--five">Get the flexibility of the cloud and the efficiency of physical servers. MAAS is designed for devops at scale, in places where bare metal is the best way to run your app. Big data, private cloud, PAAS and HPC all thrive on MAAS.</p>
-      <p><a href="https://pages.ubuntu.com/eBook-MAAS.html?source=maas_page&medium=Banner&campaign=MAAS_ebook&_ga=2.22106171.1851959601.1510602087-405342743.1460033629" class="p-link--external" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External link', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about server provisioning with MAAS' });">Learn more about server provisioning with MAAS</a></p>
+    <div class="col-6">
+      <p>You run the data center, but your customers decide what they want to do with the hardware. They love the cloud experience, but it’s more efficient for you to own the hardware.</p>
+      <p>MAAS provides super-fast self-service provisioning of Windows, Ubuntu, CentOS and ESXi.</p>
+      <div>
+        <a href="https://pages.ubuntu.com/eBook-MAAS.html?source=maas_page&medium=Banner&campaign=MAAS_ebook&_ga=2.22106171.1851959601.1510602087-405342743.1460033629" class="p-link--external" onClick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External link', 'eventAction' : 'Click', 'eventLabel' : 'Learn more about server provisioning with MAAS' });">Learn more about server provisioning with MAAS</a>
```

Comment:
This link should be in a paragraph

Reply:
Ideally in its own row

Reply:
Not sure what you mean for the second comment? Link is in a `p` now though